docs/content/common-subs/install-helm-kind.md

```diff
@@ -0,0 +1,14 @@
+<!--install-helm-kind-start-->
+```shell hl_lines="6"
+KUBECONFIG=~/.kube/config kubectl config use-context ks-core  
+kubectl create namespace kubestellar  
```

Comment:
I see that this document generally follows the pattern of `KUBECONFIG=~/.kube/config command`, and I think that is a good way to make it clear to the user that the choice of a kubeconfig file is a relevant thing to be paying attention to. It is a way of solving the problem of making context (broadly construed) visible without replacing text (very convenient and accessible) with pictures (horrible for copy-and-paste by will-sighted people and even worse for consumption by others) of sessions in shells graphically show context.

I assume that the lack of the KUBECONFIG prefix on the second command here is an oversight. As this currently stands, the second command only works if elsewhere the user has done `export KUBECONFIG=~/.kube/config` or equivalent. If that assumption were good enough for the second command here then it would also be good enough for the first.